subhub/cfg.py

```diff
@@ -377,6 +377,13 @@ def NEW_RELIC_DISTRIBUTED_TRACING_ENABLED(self):
         """
         return self("NEW_RELIC_DISTRIBUTED_TRACING_ENABLED", True)
 
+    @property
+    def ALLOWED_ORIGIN_SYSTEMS(self):
+        """
+        ALLOWED_ORIGIN_SYSTEMS
+        """
+        return [x.lower() for x in (self("ALLOWED_ORIGIN_SYSTEMS", "").split(","))]
```

Comment:
why the `.lower()`?

Reply:
I was taking case sensitivity out of the mix for callers and also application configuration for deployment agents (us).